e2e/tests/core/02-client/client_test.go

```diff
@@ -81,7 +142,7 @@ func (s *ClientTestSuite) TestClientUpdateProposal_Succeeds() {
 		subjectClientID    string
 		substituteClientID string
 		// set the trusting period to a value which will still be valid upon client creation, but invalid before the first update
-		badTrustingPeriod = time.Second * 10
+		badTrustingPeriod = time.Duration(time.Second * 10)
```

Comment:
[nit] this change is not redundant, the type is already inferred from the `time.Second * 10`